docker-widget/docker.lua

```diff
@@ -15,363 +15,406 @@ local naughty = require("naughty")
 local gears = require("gears")
 local beautiful = require("beautiful")
 
-local HOME_DIR = os.getenv("HOME")
-local WIDGET_DIR = HOME_DIR .. '/.config/awesome/awesome-wm-widgets/docker-widget'
-local ICONS_DIR = WIDGET_DIR .. '/icons/'
+--- Utility Function to handle alternative paths
+local function script_path()
+	local str = debug.getinfo(2, "S").source:sub(2)
+	return str:match("(.*/)")
+end
+
+local WIDGET_DIR = script_path()
+local ICONS_DIR = WIDGET_DIR .. "/icons/"
+
+local clickable_container = require(WIDGET_DIR .. "clickable-container")
```

Comment:
This is not a reliable method of requiring relative to the local script
```
error: ...nfig/awesome/awesome-wm-widgets/docker-widget/docker.lua:27: module '/home/ryex/.config/awesome/awesome-wm-widgets/docker-widget/clickable-container' not found:
	no field package.preload['/home/ryex/.config/awesome/awesome-wm-widgets/docker-widget/clickable-container']
	no file '/usr/share/lua/5.4//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container.lua'
	no file '/usr/share/lua/5.4//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container/init.lua'
	no file '/usr/lib/lua/5.4//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container.lua'
	no file '/usr/lib/lua/5.4//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container/init.lua'
	no file './/home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container.lua'
	no file './/home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container/init.lua'
	no file '/home/ryex/.config/awesome//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container.lua'
	no file '/home/ryex/.config/awesome//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container/init.lua'
	no file '/etc/xdg/awesome//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container.lua'
	no file '/etc/xdg/awesome//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container/init.lua'
	no file '/usr/share/awesome/lib//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container.lua'
	no file '/usr/share/awesome/lib//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container/init.lua'
	no file '/usr/lib/lua/5.4//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container.so'
	no file '/usr/lib/lua/5.4/loadall.so'
	no file './/home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container.so'
	no file '/home/ryex/.config/awesome//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container.so'
	no file '/etc/xdg/awesome//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container.so'
	no file '/usr/share/awesome/lib//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container.so'
	no file '/home/ryex/.config/awesome/modules/lockscreen/lib//home/ryex//config/awesome/awesome-wm-widgets/docker-widget/clickable-container.so'
	no file ''
	no file '/usr/lib/lua/5.4//home/ryex/.so'
	no file '/usr/lib/lua/5.4/loadall.so'
	no file './/home/ryex/.so'
	no file '/home/ryex/.config/awesome//home/ryex/.so'
	no file '/etc/xdg/awesome//home/ryex/.so'
	no file '/usr/share/awesome/lib//home/ryex/.so'
	no file '/home/ryex/.config/awesome/modules/lockscreen/lib//home/ryex/.so'
	no file ''
```

perhaps try
```lua
local clickable_container = require((...):match("(.-)[^%.]+$") .. "clickable-container")
```